comp/dogstatsd/server/server.go

```diff
@@ -773,6 +773,17 @@ func (s *server) parseMetricMessage(metricSamples []metrics.MetricSample, parser
 		} else {
 			metricSamples[idx].Tags = metricSamples[0].Tags
 		}
+
+		// If we're in serverless mode, we need to determine the metric source from extra tags
+		if s.enrichConfig.serverlessMode {
+			isRuntime := strings.HasPrefix(metricSamples[idx].Name, "runtime.")
```

Comment:
There's a case where prefix is `.jvm` for `java` right?

Reply:
There's an existing case for `jvm.*` metrics that I don't think is necessary to cover again: https://github.com/DataDog/datadog-agent/pull/35844/files#diff-a16207429aa26872091b358ada19dbad52797ca17561c5a5eee4fbacd5518315R71-R73